src/test/controller/BaseController.ts

```diff
@@ -85,7 +88,9 @@ export class BaseController {
       // }
       null
     );
-    return mockSdk.auth();
+    const firebaseAdmin = mockSdk.auth();
+    (firebaseAdmin as any).autoFlush();
+    return firebaseAdmin;
```

Comment:
Why did you make this change?

Reply:
Ohya i was thinking its too much for developers to remember to call auto Flush for tests when they use mock firebase admin. In my FirebaseService.ts class, notice how I only need to call mockFirebaseAdmin() without needing to remember to call autoFlush. Originally, I forgot to call autoFlush and my tests were timing out. 

Would this be okay?

Reply:
Oh I see, thanks!